common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +54,25 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    def test_logger_latin1_characters(self):
```

Comment:
You can also reduce the scope of test, I really just want to make sure that the context is being set properly.